R/status.R

```diff
@@ -3,31 +3,45 @@
 #' Overview of the current status of the laminr package and its dependencies.
 #' Can be useful for debugging.
 #'
-#' @returns Prints details of the current laminr status
+#' @returns A `laminr_status` object
 #' @export
 #'
 #' @details
 #' Provides information that can be useful for debugging. To run the function
-#' when an error occurs set `options(error = laminr::laminr_status)`. Note that
+#' when an error occurs set
+#' `options(error = error = \() {print(laminr::laminr_status()))`. Note that
```

Comment:
does this work?

```suggestion
#' when an error occurs, set
#' `options(error = \() { print(laminr::laminr_status() })`. Note that
```

Also is it worth using the more traditional function notation?

```suggestion
#' when an error occurs, set
#' `options(error = function() { print(laminr::laminr_status() })`. Note that
```